src/chrome/content/rules/444.hu.xml

```diff
@@ -1,15 +1,6 @@
-<!--
-	Nonfunctional domains:
-	- 4cdn.hu (image assets)
-	- img.444.hu (refused)
-	- q.444.hu (expired)
-	- cimlap.444.hu (expired)
```

Comment:
Why did you removed those hosts? They still exist.

Reply:
HTTPS is now correctly configured for these domains as well.

Reply:
So they should be added as targets.